src/main/java/frc/robot/utilities/SendableSparkMAXPID.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.utilities;
+
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+
+public class SendableSparkMAXPID implements Sendable {
+
+  private final SparkPIDController sparkPID;
+
+  public SendableSparkMAXPID(SparkPIDController pid) {
+    sparkPID = pid;
+  }
+
+  @Override
+  public void initSendable(SendableBuilder builder) {
+    builder.addDoubleProperty("kP", sparkPID::getP, sparkPID::setP);
+    builder.addDoubleProperty("ki", sparkPID::getI, sparkPID::setI);
+    builder.addDoubleProperty("kd", sparkPID::getD, sparkPID::setD);
+    throw new UnsupportedOperationException("Unimplemented method 'initSendable'");
```

Comment:
Looks implemented to me. Though I'd add something to cache the PID values like in CANSparkMaxSim class, so it doesn't add a bunch of extra CAN activity updating the Shuffleboard widget. 

Plus setting the widget type to PIDController is nifty